### PR TITLE
cli: Fixes for #1880 and #1874

### PR DESCRIPTION
**Pull request by jfoutts-celo**
### Description

Fetches name for validator:list and adds "--no-truncate" option.

### Tested

Tested that validator names are correctly fetched.

### Related issues

#1880 and #1874

### CODE REVIEW

File: packages/cli/src/commands/validator/list.ts [hunk 1]
```diff
@@ -7,25 +9,43 @@ export default class ValidatorList extends BaseCommand {
...
   async run() {
-    this.parse(ValidatorList)
+    const res = this.parse(ValidatorList)
 
     cli.action.start('Fetching Validators')
+    const accounts = await this.kit.contracts.getAccounts()
     const validators = await this.kit.contracts.getValidators()
     const validatorList = await validators.getRegisteredValidators()
 
+    var names = []
```
Comment (asaj) [inline packages/cli/src/commands/validator/list.ts]: nit: consider:

```
const names = await Promise.all(validatorList.map((v) => accounts.getName(v.address)))

File: packages/cli/src/commands/validator/list.ts [hunk 2]
```diff
@@ -7,25 +9,43 @@ export default class ValidatorList extends BaseCommand {
...
+    for (let i = 0; i < validatorList.length; i++) {
+      names.push(accounts.getName(validatorList[i].address))
+    }
+    for (let i = 0; i < validatorList.length; i++) {
+      const name = await names[i]
+      validatorList[i].name = name ? name : ''
```
Comment (asaj) [inline packages/cli/src/commands/validator/list.ts]: Would it be easier to modify the `Validators` contractkit wrapper to pull the name? If we're going to want to the name everywhere we pull validator info, that might save us from having to duplicate this code elsewhere

File: packages/cli/src/commands/validator/list.ts (hunk 1 above)
Comment (jfoutts-celo) [reply packages/cli/src/commands/validator/list.ts]: Nice!

File: packages/cli/src/commands/validator/list.ts (hunk 2 above)
Comment (jfoutts-celo) [reply packages/cli/src/commands/validator/list.ts]: Sounds good

File: packages/contractkit/src/wrappers/Validators.ts [hunk 3]
```diff
@@ -244,7 +245,14 @@ export class ValidatorsWrapper extends BaseWrapper<Validators> {
   /** Get list of registered validators */
   async getRegisteredValidators(): Promise<Validator[]> {
     const vgAddresses = await this.getRegisteredValidatorsAddresses()
-    return Promise.all(vgAddresses.map((addr) => this.getValidator(addr)))
+    let validators = await Promise.all(vgAddresses.map((addr) => this.getValidator(addr)))
+    const accounts = await this.kit.contracts.getAccounts()
+    return Promise.all(
+      validators.map(async function(v) {
+        v.name = (await accounts.getName(v.address)) || ''
```
Comment (asaj) [inline packages/contractkit/src/wrappers/Validators.ts]: How about we set this in `getValidator` instead? And please update any other CLI commands that show validators to print the name if they don't already

Comment (jfoutts-celo) [reply packages/contractkit/src/wrappers/Validators.ts]: Looks like validator:show and election:current already show name

File: packages/contractkit/src/wrappers/Validators.ts [hunk 4]
```diff
@@ -18,6 +18,7 @@ import {
 } from './BaseWrapper'
 
 export interface Validator {
+  name: string
```
Comment (mcortesi) [inline packages/contractkit/src/wrappers/Validators.ts:21]: Not sure if this is a "Validator" thing... instead what about "fetching the account" on the cli when you want to show the Validator details?

I don't know if this sounds like something any contractKit user would want, maybe just the cli.

Comment (mcortesi) [reply packages/contractkit/src/wrappers/Validators.ts:21]: hehehe.. I see that me and @asaj are thinking completely opposite things!.. 
It's not a big issue so either way work

**Review by asaj — APPROVED** [review]

File: packages/contractkit/src/wrappers/Validators.ts (hunk 3 above)
Comment (asaj) [reply packages/contractkit/src/wrappers/Validators.ts]: Awesome! Will be nice to have those populated as well "for free" 😄

**Pull request merged**